common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -32,12 +32,14 @@ $sequence--border-color: #C8C8C8;
 
 // ====================
 
-nav.sequence-nav {
+.sequence-nav {
   // TODO (cpennington): This doesn't work anymore. XModules aren't able to
   // import from external sources.
   @extend .topbar;
-  margin: -4px 0 ($baseline*1.5);
   position: relative;
+  margin: -4px 0 ($baseline*1.5);
+  padding-left: ($baseline*2);
```

Comment:
I don't want to overwrite any inherited top and bottom padding. Unless it's specified in the same set of rules, I tend to only set what I'm trying to manipulate. Is this okay, or should I find out what, if anything, is being set earlier and combine?